bundles/io/org.eclipse.smarthome.io.transport.mdns/src/main/java/org/eclipse/smarthome/io/transport/mdns/discovery/MDNSDiscoveryService.java

```diff
@@ -62,12 +62,6 @@ public void unsetMDNSClient(MDNSClient mdnsClient) {
 
     @Override
     protected void startBackgroundDiscovery() {
-        scheduler.schedule(new Runnable() {
```

Comment:
AFAIK the scan is running in the background (like in UPnPDiscovery), we only need to register listeners. MdnsClient.list() would not initiate a scan but just collect the service descriptors of all different listed services already found. Why should we do that?